babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -419,6 +419,20 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testUpdateFromTableBigQuery() {
+    final String sql = "update foo from bar set foo.x = bar.y, foo.z = bar.k";
+    final String expected = "UPDATE `foo` SET `foo`.`x` = `bar`.`y`, "
+        + "`foo`.`z` = `bar`.`k` FROM `bar`";
+    bigQuerySql(sql).ok(expected);
```

Comment:
Rather than doing it like this, is it possible to reuse this existing pattern?

https://github.com/googleinterns/calcite/blob/master/core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java#L8629

Reply:
Good idea, I didn't know that was possible.

Reply:
When I do this the none of the unparsed identifiers have back ticks surrounding them. Is this expected?

Reply:
Yeah it looks like in the test linked in the above comment, the unparsed version of that SQL string doesn't have any backticks either.